Tests/TuistGeneratorTests/Generator/LinkGeneratorTests.swift

```diff
@@ -469,6 +469,24 @@ final class LinkGeneratorErrorTests: XCTestCase {
             XCTAssertEqual($0 as? LinkGeneratorError, LinkGeneratorError.missingProduct(name: "Test"))
         }
     }
+    
+    func test_linkableDependencies_appClipSDKNode() throws {
+        // Given
+        let (pbxTarget, target) = createTargets(product: .appClip)
+        let path = AbsolutePath("/path")
+        let pbxproj = PBXProj()
+        pbxproj.add(object: pbxTarget)
+
+        let graph = Graph.create(project: .test(path: path),
+                                 dependencies: [(target: target, dependencies: [])])
+
+        // When
+        let linkableModules = try graph.linkableDependencies(path: path, name: target.name)
```

Comment:
@pepibumur this is my attempt to test inferred AppClip framework dependency for target of **appClip** product type

Reply:
Sounds good. Although I think this test should be in `GraphTests`. What do you think?

Reply:
Let me revisit :thinking: